### PR TITLE
Added Test for load_challenge function

### PR DESCRIPTION
**Pull request by nsjcorps**
#### Added test for load_challenge function

I have made the test to cover all test cases and make sure that the function does not break for any input

### CODE REVIEW

**Comment by nsjcorps** [comment]
All is good now. It works perfectly now

File: tests/unit/remoteworker/test_remote_worker.py [hunk 1]
```diff
@@ -170,3 +171,30 @@ def test_create_dir_as_python_package(self):
...
         self.assertFalse(os.path.exists(self.temp_directory))
+
+
+class LoadChallengeTestClass(BaseTestClass):
+    @mock.patch("scripts.workers.remote_submission_worker.CHALLENGE_DATA_BASE_DIR", "mocked/dir/challenge_data")
+    @mock.patch("scripts.workers.remote_submission_worker.create_dir_as_python_package")
+    @mock.patch("scripts.workers.remote_submission_worker.extract_challenge_data")
+    @mock.patch("scripts.workers.remote_submission_worker.get_challenge_phases_by_challenge_pk")
+    @mock.patch("scripts.workers.remote_submission_worker.get_challenge_by_queue_name")
+    def test_load_challenge_successfully(self, mock_get_challenge_by_queue_name, mock_get_challenge_phases, mock_extract_challenge_data, mock_create_dir):
```
Comment (Ram81) [inline tests/unit/remoteworker/test_remote_worker.py]: @nsjcorps change this to `test_load_challenge_success`

**Comment by nsjcorps** [comment]
@Ram81 , @RishabhJain2018 , @yashdusing , @KhalidRmb , @pushkalkatara Please review.

Merry Christmas Everyone

**Comment by nsjcorps** [comment]
It is there. Under it

**Comment by nsjcorps** [comment]
That is what made the test coverage 100%

**Comment by nsjcorps** [comment]
It raises an exception which is then tested

**Comment by nsjcorps** [comment]
@Ram81

**Comment by nsjcorps** [comment]
Thanks

**Pull request closed without merging**